cmd/devp2p/internal/ethtest/snap.go

```diff
@@ -209,26 +209,28 @@ The server should return the second account of the state as the first item.`,
 		},
 
 		// The genesis stateroot (we expect it to not be served)
-		{
-			nBytes:       4000,
-			root:         s.chain.RootAt(0),
-			startingHash: zero,
-			limitHash:    ffHash,
-			expAccounts:  0,
-			expFirst:     zero,
-			expLast:      zero,
-			desc: `This test requests data at the state root of the genesis block. We expect the
-server to return no data because genesis is older than 127 blocks.`,
-		},
+		// TODO: We don't yet have a separate snapshot structure so we cannot distinguish between older states
```

Comment:
Do we have an Issue for this?

Reply:
Linked https://github.com/lambdaclass/lambda_ethereum_rust/issues/1011